Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FlameGraphs"
 uuid = "08572546-2f56-4bcf-ba4e-bab62c3a3f89"
 authors = ["Tim Holy <tim.holy@gmail.com>"]
-version = "0.2.10"
+version = "0.3.0"
```

Comment:
If we're doing a minor bump, should we just jump straight to 1.0?

Reply:
Sounds good to me 👍